.gitignore

```diff
@@ -1,6 +1,7 @@
 *.iml
 .gradle
 /local.properties
+/.idea
```

Comment:
@vladcipariu91 @tomermandelfw The hole `.idea` folder should not be removed. This folder has useful files for the project, like codestyles and similar. Please, revert this change and add any relevant exclusion like the chaches and libraries bellow.

Reply:
ok

Reply:
@igorcferreira done.